src/services/checkout.ts

```diff
@@ -109,25 +109,25 @@ class Checkout extends ApiKeyAuthenticatedService {
         );
     }
 
-    public paymentLinks(paymentLinkRequest: CreatePaymentLinkRequest): Promise<PaymentLinkResource> {
-        return getJsonResponse<CreatePaymentLinkRequest, PaymentLinkResource>(
+    public paymentLinks(paymentLinkRequest: CreatePaymentLinkRequest): Promise<PaymentLinkResponse> {
```

Comment:
This model `PaymentLinkResource` is getting replaced by `PaymentLinkResponse` which breaks backwards compatibility right?

Reply:
Good question. It's the same model, the name is just different. I guess it can break Typescript implementations indeed